presto-product-tests/README.md

```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
 They should point to your external IP address (shown by `ifconfig` on your Mac (not inside docker)).
 
+* The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap.
+We recommend at least 4GB for such profiles. You can increase docker memory by going to Docker
```

Comment:
@findepi : This fix is related to https://github.com/prestodb/presto/issues/7676
We checked for singlenode-ldap and made sure tests pass. But I see your point.
Even with singlenode profile you might need more memory depending on tests you want to run.
We recommend 4 GB as it probably works in most cases and user should increase further if needed.